ui/build/asset-manifest.json

```diff
@@ -0,0 +1,22 @@
+{
+  "files": {
+    "main.css": "/static/css/main.12cb9920.chunk.css",
+    "main.js": "/static/js/main.d6f017de.chunk.js",
+    "main.js.map": "/static/js/main.d6f017de.chunk.js.map",
+    "runtime-main.js": "/static/js/runtime-main.83c3e0c4.js",
+    "runtime-main.js.map": "/static/js/runtime-main.83c3e0c4.js.map",
+    "static/js/2.180d86fb.chunk.js": "/static/js/2.180d86fb.chunk.js",
+    "static/js/2.180d86fb.chunk.js.map": "/static/js/2.180d86fb.chunk.js.map",
+    "index.html": "/index.html",
+    "precache-manifest.16a67eff996ed758117f08eed017e554.js": "/precache-manifest.16a67eff996ed758117f08eed017e554.js",
+    "service-worker.js": "/service-worker.js",
+    "static/css/main.12cb9920.chunk.css.map": "/static/css/main.12cb9920.chunk.css.map",
+    "static/js/2.180d86fb.chunk.js.LICENSE.txt": "/static/js/2.180d86fb.chunk.js.LICENSE.txt"
+  },
+  "entrypoints": [
+    "static/js/runtime-main.83c3e0c4.js",
+    "static/js/2.180d86fb.chunk.js",
+    "static/css/main.12cb9920.chunk.css",
+    "static/js/main.d6f017de.chunk.js"
+  ]
+}
```

Comment:
This looks like it ought not to be included in this PR.

And that goes for all of the `ui/build` directory. Perhaps that directory is a candidate for the gitignore file?

Reply:
I don't know how i did this, build and node_modules are both in gitignore, but for some reason they are all being pulled in

Reply:
I was worried i somehow managed to copy my whole home directory in :sweat_smile: